### PR TITLE
Adding start page content for the docs

### PR DESCRIPTION
**Pull request by imujjwal96**
Related issue: #2 
In this repository, the raw file of various sub-projects will be added to the raw directory, with `raw/index.rst` being the raw file for the start page content.

Docs of sub-projects will be stored in `raw/SUB-PROJECT-NAME` 

## Screenshot
image

### CODE REVIEW

File: raw/index.rst [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+------------------
+
+The loklak wok is a Desktop application collecting data.
+
+Code: https://github.com/loklak/loklak_wok_desktop
+
+Documentation: http://dev.loklak.org/desktopwok/
+
+loklak APIs
+-----------
```
Comment (poonai) [inline raw/index.rst]: add hyper link to respective wrapper library

**Comment by pri22296** [comment]
@imujjwal96 The screenshot and the index.rst seems to not be in sync. They seem to have different toctrees. Can you attach a screenshot of the site built using the current index.rst

**Comment by imujjwal96** [comment]
The screenshot is in sync. The reason why you don't see the last 4 lines in the list is because the links doesn't exist for them and hence they weren't added.

File: raw/index.rst (hunk 1 above)
Comment (imujjwal96) [reply raw/index.rst]: Where should the hyper link point to? I made the raw file as it was in the issue #2

File: raw/index.rst [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+from various sources, including twitter. The server contains a search index 
+and a peer-to-peer index sharing interface. All messages are stored in an 
+elasticsearch index. An automatic deployment from the development branch at 
+GitHub is available for tests here https://loklak-server-dev.herokuapp.com
+
+Code: https://github.com/loklak/loklak_server
+
+Project Site: http://api.loklak.org
+
+Documentation: http://dev.loklak.org/server
```
Comment (pri22296) [inline raw/index.rst]: Use relative link instead of absolute for linking to documentation of subproject

File: raw/index.rst [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+
+loklak search is an Angular.js web application that provides a search 
+interface for end users of loklak data and showcases some of the possibilities 
+to use loklak data for analytics.
+
+Code: https://github.com/fossasia/loklak_search
+
+Project Site: http://loklak.net
+
+Documentation: http://dev.loklak.org/search/
```
Comment (pri22296) [inline raw/index.rst]: use relative link

File: raw/index.rst [hunk 4]
```diff
@@ -0,0 +1,128 @@
...
+
+loklak apps is a collection of small web apps that make use, analyze and 
+visualize of the data of the loklak server. loklak apps use the loklak api 
+server.
+
+Code: https://github.com/fossasia/apps.loklak.org
+
+Project Site: http://apps.loklak.org
+
+Documentation: http://dev.loklak.org/webapps/
```
Comment (pri22296) [inline raw/index.rst]: use relative link

File: raw/index.rst [hunk 5]
```diff
@@ -0,0 +1,128 @@
...
+
+loklak wok
+----------
+
+The loklak wok is an Android application collecting data.
+
+Code: https://github.com/fossasia/loklak_wok
+
+Documentation: http://dev.loklak.org/androidwok/
```
Comment (pri22296) [inline raw/index.rst]: use relative link

File: raw/index.rst [hunk 6]
```diff
@@ -0,0 +1,128 @@
...
+
+loklak wok Desktop
+------------------
+
+The loklak wok is a Desktop application collecting data.
+
+Code: https://github.com/loklak/loklak_wok_desktop
+
+Documentation: http://dev.loklak.org/desktopwok/
```
Comment (pri22296) [inline raw/index.rst]: use relative link

**Review by pri22296 — CHANGES_REQUESTED** [review]

File: raw/index.rst (hunk 2 above)
Comment (imujjwal96) [reply raw/index.rst]: I was a bit skeptical about it as these links don't actually exist right now. Maybe we can make these changes later when we have the documentation of these sub-projects in the repository.

**Review by pri22296 — APPROVED** [review]

File: raw/index.rst (hunk 2 above)
Comment (pri22296) [reply raw/index.rst]: @imujjwal96 we can make the required changes in a seperate PR after [#1127](https://github.com/loklak/loklak_server/pull/1127) gets merged.

Comment (imujjwal96) [reply raw/index.rst]: :+1:

File: raw/index.rst (hunk 1 above)
Comment (poonai) [reply raw/index.rst]: each library having their own repo https://github.com/loklak/loklak_go_api

Comment (imujjwal96) [reply raw/index.rst]: But then these APIs ought to have their own documentation. We can at a later stage, when their documentations are integrated, add links to both their github repository and their documentation the way we did with the other repositories.

Comment (imujjwal96) [reply raw/index.rst]: I think our primary focus at the moment should be to integrate everything. Such minor changes can me made afterwards.

Comment (poonai) [reply raw/index.rst]: okay cool

**Review by poonai — APPROVED** [review]

**Comment by imujjwal96** [comment]
@mariobehling Still waiting for @yukiisbored's review

**Review by yukiisbored — CHANGES_REQUESTED** [review]
Please use relative links so it'll work locally, under any domain name, on someone else's computer, on my phone locally, etc.

**Review by pri22296 — COMMENTED** [review]
use rst directive `:doc:` to link to the specific `index.rst` file within subprojects.

**Comment by imujjwal96** [comment]
@pri22296 I'm sorry, but I don't exactly follow what I should change. Can you please elaborate it more?

**Comment by pri22296** [comment]
@imujjwal96 use ``:doc:`DOCUMENTATION <SUB-PROJECT/index>` `` instead of `` `DOCUMENTATION <SUB-PROJECT>`_``

**Comment by pri22296** [comment]
@imujjwal96 Please squash the commits. other than that LGTM

**Review by pri22296 — APPROVED** [review]

**Comment by yukiisbored** [comment]
Please move it into `index.rst.d` and rename it to `00_readme.rst`

**Pull request merged**